lib/wwwtxt/plain_text_renderer.rb

```diff
@@ -0,0 +1,123 @@
+require "htmlentities"
+
+module WwwTxt
+  class PlainTextRenderer
+    FORMAT = "txt"
+
+    def self.render_from_html(html)
+      strip_html(html)
+    end
```

Comment:
Ah you'll also need an `#initialize` setting those arguments and whatnot too. 

If you'd rather keep the class method thing going, this is more like a module than a class, in which case you could make it a module and simply use `strip_html` via [module_function(http://ruby-doc.org/core-2.2.0/Module.html#method-i-module_function)